### PR TITLE
POCAMRS-506: Create patient alerts microfrontend

### PR DESCRIPTION
**Pull request by denniskigen**
[Ticket](https://jira.ampath.or.ke/browse/POCAMRS-506)

Create a patient alerts microfrontend that handles fetching of reminders from our ETL backend, renders them as toasts and displays those in the notifications menu panel. This PR also configures the notifications menu button on the navbar. When alerts are present, the `NotificationNew` icon is displayed while if alerts are not present, just the `Notification` icon is shown.

Screenshot 2021-07-10 at 13 19 15

Screenshot 2021-07-11 at 17 39 24

### CODE REVIEW

**Comment by jonathandick** [comment]
@denniskigen this LGTM.But one thing missing is the "actions". Right now in our alerts, it's possible to enroll into a program, etc.

File: packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          onClick={() => togglePanel('notificationsMenu')}>
+          {isActivePanel('notificationsMenu') ? (
+            <Close20 />
+          ) : alerts.length && hasUnreadAlerts ? (
+            <NotificationNew20 className={styles.unreadNotifications} />
+          ) : (
+            <Notification20 />
+          )}
+        </HeaderGlobalAction>
+      ) : null}
```
Comment (donaldkibet) [inline packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx:35]:
```suggestion
      {alerts && (
        <HeaderGlobalAction
          aria-label="Notifications"
          aria-labelledby="Notifications Icon"
          name="Notifications"
          isActive={isActivePanel('notificationsMenu')}
          onClick={() => togglePanel('notificationsMenu')}>
          {isActivePanel('notificationsMenu') ? (
            <Close20 />
          ) : alerts.length && hasUnreadAlerts ? (
            <NotificationNew20 className={styles.unreadNotifications} />
          ) : (
            <Notification20 />
          )}
        </HeaderGlobalAction>
      )}
```

**Review by donaldkibet — APPROVED** [review]
LTGM with few `nit` suggestion

File: packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx:35]: Fair enough, though like using ternaries to be explicit.

**Comment by denniskigen** [comment]
> @denniskigen this LGTM.But one thing missing is the "actions". Right now in our alerts, it's possible to enroll into a program, etc.

About actions, can I work on that as an enhancement in a follow-up PR?

**Comment by jonathandick** [comment]
> > @denniskigen this LGTM.But one thing missing is the "actions". Right now in our alerts, it's possible to enroll into a program, etc.
> 
> About actions, can I work on that as an enhancement in a follow-up PR?

Sure. Let's get this merged.

File: packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx (hunk 1 above)
Comment (donaldkibet) [reply packages/esm-patient-alerts-app/src/notifications-menu-button.component.tsx]: I will go ahead and merge this in

**Pull request merged**